### ISSUE TITLE
Error [segmentation fault core dumped] during training act

### ISSUE THREAD

**Issue by haoyangzhanglab** [OpenTeleVision/TeleVision#19]
Hi, I am trying to reproduce the training process of "00-can-sorting". However, when following command in readme.md, some error happened. After some testing via **"line by line printing"**, I think there is something wrong with code `xs = self[0](tensor_list)` in `backbone.py` . Here is some information.
Command:
`python imitate_episodes.py --policy_class ACT --kl_weight 10 --chunk_size 60 --hidden_dim 512 --batch_size 45 --dim_feedforward 3200 --num_epochs 50000 --lr 5e-5 --seed 0 --taskid 00 --exptid 01-sample-expt`
My testing code:
![image](https://github.com/user-attachments/assets/28edd56d-4936-4a99-bd59-3391b132271d)
Output:
![image](https://github.com/user-attachments/assets/13918e96-42fc-436b-93d4-a4df832be0ba)
I have no idea about solving this bug, could you help me? Thanks!

**Comment by haoyangzhanglab** [comment]
Sorry for bother. Just some env error with torch
